pontoon/translate/static/js/translate.es6

```diff
@@ -0,0 +1,499 @@
+/**
+ * Model for Entities, which represent a single string to be translated.
+ */
+class Entity {
+  constructor(data) {
+    $.extend(this, data);
```

Comment:
This kind of "overwrite everything" idea smells funny to me. It makes me think about security problems. I'd either explicitly name fields, or have something like `this.data = data;`.